android/app/src/main/res/raw/keep.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--Search categories translations/keys are loaded from C++, explicitly keep them to avoid stripping by R8-->
+<!-- Explicitly keep all string resources as the R8 shrinker might discard some of them as unused,
```

Comment:
Does it keep only our jar resources, or also other libs resources? What is the diff with and without this file?

Reply:
Does it mean that we should revert my previous patch?

Reply:
The previous patch is useful still, its good to have a common prefix for related strings.

Reply:
> Does it keep only our jar resources, or also other libs resources? What is the diff with and without this file?

Libs' string resources are kept as well, e.g. there were 7 unused `abc_`-prefixed strings that came from the android appcompat lib. They were dropped before, but are included now. There seems to be no way to apply different shrink settings to included libs.

Only the `resources.arsc` file is different in my test builds apks.